packages/autoloader/src/AutoloadGenerator.php

```diff
@@ -84,15 +83,14 @@ public function dump(
 		$fileMap  = $this->getFileMap( $autoloads, $filesystem, $vendorPath, $basePath );
 
 		// Generate the files.
-		file_put_contents( $targetDir . '/autoload_classmap_package.php', $this->getAutoloadClassmapPackagesFile( $classMap ) );
-		$this->io->writeError( '<info>Generated ' . $targetDir . '/autoload_classmap_package.php</info>', true );
+		file_put_contents( $targetDir . '/autoload_classmap.php', $this->getAutoloadClassmapPackagesFile( $classMap ) );
```

Comment:
A file named `autoload_classmap.php` is also generated by composer. Here's an [example](https://plugins.trac.wordpress.org/browser/jetpack/trunk/vendor/composer/autoload_classmap.php). We have been leaving all of the composer generated files alone, so I’m not sure that we want to overwrite this one. Maybe we could name it something like `jp_autoloader_classmap.php` instead?